### PR TITLE
Convert to Gradle project with native component and add native-image support

### PR DESCRIPTION
**Pull request by kaneeldias**
## Purpose
$subject

Related issue: https://github.com/ballerina-platform/ballerina-standard-library/issues/3692

### CODE REVIEW

**Review by niveathika — DISMISSED** [review]
LGTM. Do change Slack mentions to Discord

File: ballerina/Module.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+
+## Usage
+
+To add the PostgreSQL driver dependency the project, simply import the module as below,
+
+```ballerina
+import ballerina/sql;
+import ballerinax/postgresql;
+import ballerinax/postgresql.driver as _;
+```
```
Comment (niveathika) [inline ballerina/Module.md:23]: Add useful links

**Review by daneshk — APPROVED** [review]

File: ballerina/Module.md (hunk 1 above)
Comment (kaneeldias) [reply ballerina/Module.md:23]: We usually don't have the Useful links section in Module.md. Instead we have them in Package.md

e.g.
https://github.com/ballerina-platform/module-ballerina-io/blob/master/ballerina/Module.md
https://github.com/ballerina-platform/module-ballerina-io/blob/master/ballerina/Package.md

Comment (kaneeldias) [reply ballerina/Module.md:23]: We usually don't have the Useful links section in Module.md. Instead we have them in Package.md

e.g.
https://github.com/ballerina-platform/module-ballerina-io/blob/master/ballerina/Module.md
https://github.com/ballerina-platform/module-ballerina-io/blob/master/ballerina/Package.md

**Review by niveathika — APPROVED** [review]

**Pull request merged**